Multisig/UI/Transactions/TransactionDetails/AdvancedTransactionDetailsView.swift

```diff
@@ -9,11 +9,39 @@
 import SwiftUI
 
 struct AdvancedTransactionDetailsView: View {
-    let transaction: TransactionViewModel
+    @ObservedObject
+    var theme: Theme = App.shared.theme
     
+    let transaction: TransactionViewModel
+    private let padding: CGFloat = 11
+
     var body: some View {
-        Text("Hello, World!").onAppear {
+        List {
+            if transaction.nonce != nil {
+                KeyValueRow("Nonce", transaction.nonce!, true, .gnoDarkGrey).padding(.vertical, padding)
+            }
+
+            if transaction.operation != nil {
+                KeyValueRow("Type of operation", transaction.operation!, true, .gnoDarkGrey).padding(.vertical, padding)
+            }
+
+            if transaction.hash != nil {
+                KeyValueRow("Transaction hash", transaction.hash!, true, .gnoDarkGrey).padding(.vertical, padding)
```

Comment:
with 4 parameters we need parameter labels, can you add them, please? Something like key, value, copyEnabled, textColor